src/deployment-guides/TKO-TO-Customized-Dashboards.md

```diff
@@ -0,0 +1,115 @@
+﻿
+# Customize Tanzu Observability Dashboard for Tanzu for Kubernetes Operations
+
+Tanzu Observability provides various out of the box dashboards.We have customized the provided dashboards for your particular deployment.Customized dashboards monitors the following Tanzu for Kubernetes Operations components:
```

Comment:
... various out-of-the-box dashboards. You can customize the dashboards for your particular deployment. This document provides the steps to customize the dashboards to monitor the following Tanzu for Kubernetes Operations components: